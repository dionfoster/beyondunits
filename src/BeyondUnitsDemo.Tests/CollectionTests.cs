using ApprovalTests;
using ApprovalTests.Reporters;
using BeyondUnitsDemo.Tests.Helpers;
using NUnit.Framework;

namespace BeyondUnitsDemo.Tests
{
    [TestFixture]
    public class CollectionTests
    {
        [Test]
        [UseReporter(typeof(DiffReporter))]
        public void People_are_ordered_by_firstname()
        {
            var people = new Data().GetPeopleOrderedByFirstName();

            var report = new MethodCalleeCollectionApprovalReport<Person>
                             {
                                 Body = people,
                                 BodyFormatter = o => string.Format("{0} {1}", o.Firstname, o.Surname)
                             };
            
            Approvals.Verify(report);
        }
    }
}
