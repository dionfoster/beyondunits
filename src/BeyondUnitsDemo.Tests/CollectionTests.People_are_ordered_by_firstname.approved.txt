===============================
People are ordered by firstname
===============================
Aaron Smith
Barry Todd
Callum MacDonald
Tom Tunstal
